server/channels/app/permissions_migrations.go

```diff
@@ -614,17 +614,12 @@ func (a *App) getAddExperimentalSubsectionPermissions() (permissionsMap, error)
 		// Give the new subsection READ permissions to any user with READ_EXPERIMENTAL
 		permissionTransformation{
 			On:  permissionExists(model.PermissionSysconsoleReadExperimental.Id),
-			Add: []string{model.PermissionSysconsoleReadExperimentalBleve.Id, model.PermissionSysconsoleReadExperimentalFeatures.Id, model.PermissionSysconsoleReadExperimentalFeatureFlags.Id},
+			Add: []string{model.PermissionSysconsoleReadExperimentalFeatures.Id, model.PermissionSysconsoleReadExperimentalFeatureFlags.Id},
```

Comment:
Should we consider removing any existing bleve permissions currently assigned to users?

Reply:
Don't think we need to go to that level. It's okay to just let them be.